content/post/097-js-ipfs-0.46.md

```diff
@@ -0,0 +1,161 @@
+---
+date: 2020-06-05
+url: /2020-06-05-js-ipfs-0-46/
+title: js-ipfs 0.46.0 released
+header_image: js-ipfs-placeholder.png
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> Bitswap 1.2.0, transfer speed improvements & go-IPFS 0.5.x compatibility
+
+Blink and you’ll miss it, ~~`js-IPFS@0.45.0`~~ `js-IPFS@0.46.0` has been released; goodies include Bitswap 1.2.0, much faster file transfers, and go-IPFS 0.5.x compatibility!
+
+## ↔️ Bitswap 1.2.0
+
+Version 1.2.0 of the Bitswap protocol has landed in `js-IPFS` and brings a host of performance improvements around coordinating the supply of blocks to peers across the network. For example, peers can now respond to WANTs with messages saying they have the block but not sending it, in order to later send multiple blocks in one message instead of many small messages resulting in less chatty network operations and faster overall transfer times.
```

Comment:
It's `js-ipfs` in the title, but `js-IPFS` here. I prefer all lower... but IIRC we decided on a convention one way or the other a while back, right?

Reply:
Uff, I may have missed that conversation - I'll make it uniform, what did we decide?

Reply:
I've made it `js-IPFS`, let me know if it's better the other way.